README.md

```diff
@@ -1,6 +1,15 @@
 # TasteIT
-A recipe bank with a scaling function.
+A recipe bank for the IT-section in Chalmers.
 
-This repo mostly contains `good first issues` which makes it a good starting point for any one interested in helping out digIT develop their IT services.
+## How to start the webpage
+1. Start by cloning down the project in a separate fork or branch
+2. Start the backend
+3. Start the frontend, this will open the webpage on [localhost:3000](http://localhost:3000/)
 
-See project page: https://github.com/cthit/TasteIT/projects/1
+### How to start backend
+2. Install dependencies by running `npm install` in the **backend** directory, this requires npm to be installed
```

Comment:
Starts with step two?

Reply:
Missed it, thanks for pointing this out!